python/whylogs/api/writer/whylabs.py

```diff
@@ -68,6 +68,16 @@
 _S3_PUBLIC_DOMAIN = os.environ.get("_WHYLABS_PRIVATE_S3_DOMAIN") or _US_WEST2_DOMAIN
 _WHYLABS_SKIP_CONFIG_READ = os.environ.get("_WHYLABS_SKIP_CONFIG_READ") or False
 
+KNOWN_CUSTOM_PERFORMANCE_METRICS = {
```

Comment:
we might call these known ranking metric names, but lets follow up and make these read form profile metadata, and mark the metadata when the ranking metrics are computed?